test/packages-src/bonjour/manifest.mf

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Is it worth keeping this in sources as it is also contained by the bonjour-v1.pkg file, and it is not used as a standalone file by tests ?

Reply:
I think so based on 2 reasons:
1. with the source code, it make us easier to make further changes in the pkg. 
2. we can use the source as a dropin in the test as well.